README.md

```diff
@@ -1,2 +1,69 @@
-# enhancements
-Enhancement Tracking and Backlog Repo for Istio Releases
+# Enhancement Tracking and Backlog
+Enhancement Tracking and Backlog Repo for Istio Releases.
+
+This repo contains issues and Istio enhancement proposals. These issues are umbrellas for new enhancements to be added to Istio. An enhancement usually takes multiple releases to complete. And an enhancement can be tracked as backlog items before work begins. An enhancement may be filed once there is consensus in at least one Istio working group.
```

Comment:
After multiple passes, I don't think this repo will contain Istio Enhancement Proposals (IEPs). It will contain the issue that documents a design doc in Google Docs.

Reply:
Agreed, that is the initial intention.  There is also some preference (cc @jdumars) to document the approved RFC is github (but not the iterations + comments).